.gitignore

```diff
@@ -15,3 +15,4 @@ spec/reports
 test/tmp
 test/version_tmp
 tmp
+example/.powenv
```

Comment:
hmm, in that case might make more sense to put in the .gitignore inside example :)